subsys/net/ip/Kconfig.stats

```diff
@@ -127,7 +127,7 @@ config NET_STATISTICS_POWER_MANAGEMENT
 
 config NET_STATISTICS_WIFI
 	bool "Wi-Fi statistics"
-	depends on NET_L2_WIFI_MGMT
```

Comment:
how about making this `depends on NET_L2_WIFI_MGMT || NET_L2_RF_WIFI_MGMT` so statistics could work with either?